product_supplierinfo_packaging/models/product_supplierinfo.py

```diff
@@ -0,0 +1,161 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+from odoo.osv.expression import FALSE_DOMAIN
+from odoo.tools.float_utils import float_is_zero, float_round
+
+
+class ProductSupplierInfo(models.Model):
+    _inherit = "product.supplierinfo"
+
+    packaging_id = fields.Many2one(
+        comodel_name="product.packaging",
+        ondelete="restrict",
+    )
+    packaging_qty = fields.Float(
+        compute="_compute_packaging_qty",
+    )
```

Comment:
Why not a related?  `related="packaging_id.qty"`

Reply:
Only for inheritance purpose